### PR TITLE
COVID19 exploration page updates

### PR DESCRIPTION
**Pull request by paulineribeyre**

### CODE REVIEW

File: qa-covid19.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -128,6 +128,7 @@
   "explorerConfig": [
     {
       "tabTitle": "Datasets",
+      "charts": {},
```
Comment (mfshao) [inline qa-covid19.planx-pla.net/portal/gitops.json:131]: Why this empty block?

File: qa-covid19.planx-pla.net/portal/gitops.json [hunk 2]
```diff
@@ -157,8 +158,7 @@
         "dataType": "dataset",
         "nodeCountTitle": "Datasets",
         "fieldMapping": [
-          { "field": "name", "name": "Dataset" },
```
Comment (mfshao) [inline qa-covid19.planx-pla.net/portal/gitops.json:160]: Should keep this field mapping but change to Dataset Name for consistency?

File: qa-covid19.planx-pla.net/portal/gitops.json (hunk 1 above)
Comment (paulineribeyre) [reply qa-covid19.planx-pla.net/portal/gitops.json:131]: if not specified, there is a console.error about GuppyDataExplorer expecting a prop and getting undefined

File: qa-covid19.planx-pla.net/portal/gitops.json (hunk 2 above)
Comment (paulineribeyre) [reply qa-covid19.planx-pla.net/portal/gitops.json:160]: in "Datasets" tab i am naming fields "Name, Source" and in other tabs i am naming them "Dataset Name, Dataset Source"

File: qa-covid19.planx-pla.net/portal/gitops.json (hunk 1 above)
Comment (mfshao) [reply qa-covid19.planx-pla.net/portal/gitops.json:131]: Oops... should fix that. Thanks

File: qa-covid19.planx-pla.net/portal/gitops.json (hunk 2 above)
Comment (mfshao) [reply qa-covid19.planx-pla.net/portal/gitops.json:160]: Thanks for the explanation

Comment (paulineribeyre) [reply qa-covid19.planx-pla.net/portal/gitops.json:160]: thanks for the thorough review :D

**Pull request merged**